clients/client/src/main/java/org/projectnessie/client/StreamingUtil.java

```diff
@@ -66,8 +66,8 @@ private StreamingUtil() {
    */
   public static Stream<CommitMeta> getCommitLogStream(@NotNull TreeApi treeApi, @NotNull String ref,
```

Comment:
This method should get the new parameters as well.

Reply:
another reason to use a wrapper object

Reply:
agreed. I'll switch stuff to using `@BeanParam` and putting all parameters into a `CommitLogParams` class